eval.go

```diff
@@ -154,6 +154,9 @@ func (o *SearchOptions) SetDefaults() {
 	if o.TotalMaxImportantMatch == 0 {
 		o.TotalMaxImportantMatch = 10 * o.ShardMaxImportantMatch
 	}
+	if o.NumContextLines > 5 {
+		o.NumContextLines = 5
+	}
```

Comment:
this isn't enforcing a default, but clamping. I also am not sure why you would ignore what the client asks for. What is the motivation? I think if you are worried about abuse, the more appropriate place for this is in the http layer and returning a badrequest response than silently clamping.